### DISCUSSION TITLE
'latest' release redirect doesn't work for exe files

### DISCUSSION THREAD

**Discussion by shoeletz** [community/community#114788] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I just created a release and I'm able to directly download a file using the "latest" redirect:
[https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost](https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost)
but when downloading the similarly named ".exe" file I get a 404 Not Found.
[https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost.exe](https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost.exe)

Any suggestions? Should I be using a different download link? Both work fine if linking directly to the release, but I'd like to link to latest so that it can be used in automation.

**Comment by ethanwater** [comment]
The name of the exe is invalid. Fix that and you're good to go.

**Reply by shoeletz** [reply]
> The name of the exe is invalid. Fix that and you're good to go.

How is it invalid? Using the direct link to the release version without 'latest' works fine.
[https://github.com/logship-io/logship-agent/releases/download/v0.3.2/Logship.Agent.ConsoleHost.exe](https://github.com/logship-io/logship-agent/releases/download/v0.3.2/Logship.Agent.ConsoleHost.exe)
vs
[https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost.exe](https://github.com/logship-io/logship-agent/releases/download/latest/Logship.Agent.ConsoleHost.exe)
